export.go

```diff
@@ -6,6 +6,14 @@ import (
 	"fmt"
 )
 
+type OrderType int
+
+// OrderTraverse is the type of traversal order to use when exporting and importing.
```

Comment:
Could we provide short reason in the Godoc which is preferred and which is backwards compatible?

Reply:
agree, what's the advantage of preorder type and why people should use it?